example/rater/generated_answer.ipynb

```diff
@@ -190,7 +193,8 @@
+      "The label2score label ['reject', 'equivalent'] not in example label.\n",
+      "RaterConfig(flow_name='RaterFlow', model_config={'model_name': 'gpt-4-1106-preview', 'model_server': 'OpenAIModelServer', 'num_call': 1, 'temperature': 0.0, 'response_format': {'type': 'json_object'}}, label2score={'accept': 1.0, 'equivalent': 0.0, 'reject': -1.0}, guided_prompt_template=GuidedPrompt(instruction=\"\\n            Compare two answers: a generated answer and a grounding answer based on a provided context and question.\\n            There are few annotated examples below, consisting of context, question, grounding answer, generated answer, explanation and label.\\n            If generated answer is better, you should give a higher score and vise versa. Check label to score dictionary: [('accept', 1.0), ('equivalent', 0.0), ('reject', -1.0)].\\n            Your response should only focus on the unlabeled sample, including two fields: explanation and label (one of ['accept', 'equivalent', 'reject']).\\n            \", examples=[Context(context='Early computers were built to perform a series of single tasks, like a calculator.', question='Did early computers function like modern calculators?', grounding_answer='No. Early computers were used primarily for complex calculating.', generated_answer='Yes. Early computers were built to perform a series of single tasks, similar to a calculator.', explanation='The generated answer is better because it correctly figures out early computers was used to perform single tasks akin to calculators and asigned a score of 1.0.', label='accept')]), num_thread=1)\n"
```

Comment:
is this text auto printed? can we avoid printing them?

Reply:
The `RaterServer.__init__()` method has a line `print(self._config)`.